tests/test_outcome_response.py

```diff
@@ -3,7 +3,7 @@
 import mock
 import unittest
 
-RESPONSE_XML = """<?xml version="1.0" encoding="UTF-8"?>
+RESPONSE_XML = """
```

Comment:
Going along with my other comment, I'd want this `RESPONSE_XML` to be a bytestring (with the `b` prefix), but otherwise left alone. That way it's semantically equivalent in Python 2.